### PR TITLE
Recipe for lorikeet

### PR DESCRIPTION
**Pull request by pvanheus**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/lorikeet/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+cp lorikeet-$PKG_VERSION.jar ${PREFIX}/bin
+
+cat <<END >${PREFIX}/bin/lorikeet
+#!/bin/sh
+
+JAR_PATH=`dirname \`which conda\``
+java ${JAVA_OPTS:-} -jar \$JAR_PATH/lorikeet-${PKG_VERSION}.jar \$*
```
Comment (bgruening) [inline recipes/lorikeet/build.sh]: please follow the guidelines for Java packages: https://bioconda.github.io/guidelines.html#java

File: recipes/lorikeet/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+about:
+  home: https://github.com/AbeelLab/lorikeet
+  summary: 'Tool for digital spoligotyping of MTB strains from Illumina read data'
+  license_family: GPL
+  license: GPL-3
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
```
Comment (bgruening) [inline recipes/lorikeet/meta.yaml]: can you please add the DOI to the recipe: 10.1371/journal.pmed.1001880

Comment (pvanheus) [reply recipes/lorikeet/meta.yaml]: DOI added. Somewhat peculiar - the paper is mentioned on the Lorikeet Github page but there is no mentioned of lorikeet nor of digital spoligotyping in the paper. Still, thanks, it is the correct DOI.

**Comment by bgruening** [comment]
Thanks Peter.

**Pull request merged**